jsonrpc-specs/src/methods/discv5.json

```diff
@@ -0,0 +1,568 @@
+		"params": [
+			{
+				"name": "enr",
+				"required": true,
+				"schema": {
+					"title": "Ethereum node record",
+					"$ref": "#/components/schemas/Enr"
+				}
+			},
+			{
+				"name": "payload",
+				"required": true,
+				"description": "request element of message-data",
+				"schema": {
+					"$ref": "#/components/schemas/hexString"
+				}
+			},
+			{
+				"name": "requestId",
+				"description": "request-id element of message-data",
+				"required": true,
+				"schema": {
+					"description": "Hex encoded RLP byte array of length <= 8 bytes",
+					"$ref": "#/components/schemas/bytes8"
+				}
+			}
+		],
+		"result": {
+			"name": "sendTalkResponseResult",
+			"description": "Returns \"true\" upon success.",
+			"schema": {
+				"type": "boolean"
+			}
+		}
+	},
+	{
+		"name": "discv5_ping",
+		"summary": "Send a PING message to the designated node and wait for a PONG response.",
+		"params": [
+			{
+				"name": "enr",
+				"required": true,
+				"schema": {
+					"title": "Ethereum node record",
+					"$ref": "#/components/schemas/Enr"
+				}
+			}
+		],
+		"result": {
+			"name": "pingResult",
+			"description": "Returns PONG response",
+			"schema": {
+				"title": "PONG message",
+				"type": "object",
+				"required": [
+                    "enrSeq",
+                    "recipientIP",
+                    "recipientPort"
+                ],
+				"properties": {
+					"enrSeq": {
+						"description": "ENR sequence number of sender",
+						"type": "number"
+					},
+					"recipientIP": {
+						"description": "IP address of the intended recipient",
+						"pattern": "((^\\s*((([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5]))\\s*$)|(^\\s*((([0-9A-Fa-f]{1,4}:){7}([0-9A-Fa-f]{1,4}|:))|(([0-9A-Fa-f]{1,4}:){6}(:[0-9A-Fa-f]{1,4}|((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){5}(((:[0-9A-Fa-f]{1,4}){1,2})|:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){4}(((:[0-9A-Fa-f]{1,4}){1,3})|((:[0-9A-Fa-f]{1,4})?:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){3}(((:[0-9A-Fa-f]{1,4}){1,4})|((:[0-9A-Fa-f]{1,4}){0,2}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){2}(((:[0-9A-Fa-f]{1,4}){1,5})|((:[0-9A-Fa-f]{1,4}){0,3}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){1}(((:[0-9A-Fa-f]{1,4}){1,6})|((:[0-9A-Fa-f]{1,4}){0,4}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(:(((:[0-9A-Fa-f]{1,4}){1,7})|((:[0-9A-Fa-f]{1,4}){0,5}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:)))(%.+)?\\s*$))"
+					},
+					"recipientPort": {
+						"description": "recipient UDP port",
+						"type": "number"
```

Comment:
Can we apply additional constraints on the port number to specify the integer range that is valid (since it is only valid for the port number to be in the range `0-65535`)